bothub/api/routers.py

```diff
@@ -19,3 +23,9 @@
 router.register('example', RepositoryExampleViewSet)
 router.register('entity/new', NewRepositoryExampleEntityViewSet)
 router.register('entity', RepositoryExampleEntityViewSet)
+router.register('translateexample', NewRepositoryTranslatedExampleViewSet)
```

Comment:
I'll just convert two words urls into snake case with underscores.

Reply:
We can use hyphens.

According [Google Webmasters](https://support.google.com/webmasters/answer/76329?hl=en) support:
Consider using punctuation in your URLs. The URL http://www.example.com/g'reen-dress.html is much more useful to us than http://www.example.com/greendress.html. We recommend that you use hyphens (-) instead of underscores (_) in your URLs.

Reply:
Perfect, that would be great